tests/lsp_features/semantic_tokens.zig

```diff
@@ -2020,8 +1970,9 @@ test "weird code" {
         .{ "bar", .function, .{ .declaration = true } },
     });
     try testSemanticTokens(
-        \\_ = async bar
+        \\_ = bar
     , &.{
+        .{ "=", .operator, .{} },
         .{ "bar", .variable, .{} },
     });
```

Comment:
This test is meant to reproduce a specific issue in the parser.  See 244f2833cd053d74ca2d3a48f2f3ee01c1cd57fe

The new version doesn't reproduce it anymore so the test should be removed.

Reply:
Removed the test in https://github.com/zigtools/zls/pull/2394/commits/398991fecc70a67c4de52d332d6274fbf717c742.